.gitignore

```diff
@@ -0,0 +1,75 @@
+.metadata
```

Comment:
You need to edit gitignore file, because you have in repository many unused files